pitest-entry/src/main/java/org/pitest/util/FileUtil.java

```diff
@@ -24,7 +25,7 @@ public static String readToString(final InputStream is)
 
   public static String randomFilename() {
     return System.currentTimeMillis()
-        + ("" + Math.random()).replaceAll("\\.", "");
+        + ("" + ThreadLocalRandom.current().nextDouble()).replaceAll("\\.", "");
```

Comment:
Figured contextualize this change with a link to the recipe that made it:
- https://docs.openrewrite.org/recipes/java/migrate/util/replacemathrandomwiththreadlocalrandomrecipe
- https://github.com/openrewrite/rewrite-migrate-java/pull/543